frontend/src/scenes/insights/views/WorldMap/countryCodes.ts

```diff
@@ -510,7 +517,13 @@ export const languageCodeToName: Record<string, string> = {
 
 // This is only used as a fallback for some languages that don't usually
 // come in the locale-country format (such as nl-NL usually being presented simply as nl)
-// We'll fill this as we see fit based on the values seen in the wild
+// but that can't simply be translated to a country by capitalizing the locale
 const languageCodeToCountryCode: Record<string, string> = {
-    nl: 'NL',
+    ar: 'SA', // Arabic -> Saudi Arabia, tricky, there's no good representation for the "default Arab country"
+    af: 'ZA', // Afrikaans -> South Africa
+    ta: 'IN', // Tamil -> India
+    eu: 'ES', // Basque -> Spain, tricky, but there's no better flag than the Spanish one, we could move to custom SVGs if we wanted to solve this
+    cy: 'GB', // Welsh -> UK
```

Comment:
I'm not sure about the others but this should be 🏴󠁧󠁢󠁷󠁬󠁳󠁿

I wonder if this should map language code to flag directly, rather than language code -> country code -> flag

Reply:
Hmm, it might be easier that way, you're right, it makes the "Unknown" suggestion below easier as well

Reply:
Also, good catch with the Welsh flag, I didn't really know they had their flag as an emoji, y'all have a weird political structure, I hope you know that :)